src/main/java/com/exasol/adapter/document/DocumentAdapterDialect.java

```diff
@@ -18,6 +19,14 @@ public interface DocumentAdapterDialect {
      */
     public TableKeyFetcher getTableKeyFetcher(final ConnectionPropertiesReader connectionInformation);
 
+    /**
+     * Get a database specific {@link SchemaFetcher}.
+     *
+     * @param connectionInformation connection details
+     * @return database specific {@link SchemaFetcher}
+     */
+    public SchemaFetcher getSchemaFetcher(final ConnectionPropertiesReader connectionInformation);
```

Comment:
Should we add a note to changes files about this breaking change in the interface requiring all derived VSD dialects to implement this new required method?

Reply:
Good point. I will increment the major version...

Reply:
I decided to add a default implementation to stay backwards compatible.